play/PlayerRules/CharacterCreation/04_01_Weapons.txt

```diff
@@ -253,32 +253,136 @@ Minimum Strength 4, Reload DC 25% unless otherwise noted. Reload time 2 Actions.
 Jam range 2%
 ============================================================
 
-ECSC Mk9 MP $650 / $14
-45 m
-30%
-33 rds, 9mm Magazine / HP
-41
-
-ECSC Mk7 PDW $740 / $10
-60m
-40%
-25rds 9mm / HP
-41
-
-G&W XMP 30 $910 / $13
-65m
-35%
-30rds, .40 RMM / HP
-43
-
-Frontier 240M $1140 / $13
-With an innovative bolt design that rotates down 60 degrees after firing, the
-240M is able to retain impressive accuracy due to decreased muzzle rise, even 
-at high rates of fire.
-65m
-20%
-30rds 10mm
-44
+=====	LEVEL 1 REQUIREMENT WEAPONS	=====
+
+----ECSC Mk5 MP 	$1616
+Damage:	48
+Ranged Miss Chance:
+	3 out to 10m
+	4 out to 20m
+	5 out to 35m
+Burst Miss Chance:		+3 - 1 for every 6 Dex (starting at 5)
+Automatic Miss Chance:	+4 - 1 for every 5 Dex
+Mag size: 33 rounds of 8.0x20mm. $18 per mag
+Reload DC: 			30
+Fire modes:			Semi, burst and auto
+Burst; Auto fire rate:		3; 6
+Armor pierce level: 		1
+Reflex Modifier: 			+10
+Suppression Level:		Medium
+
+----ECSC Mk9 PDW 	$1027
+Damage:	45
+Ranged Miss Chance:
+	4 out to 10m
+	5 out to 20m
+	6 out to 40m
+Automatic Miss Chance:	+4 - 1 for every 5 Dex
+Mag size: 25 rounds of 9.0x16mm. $12 per mag
+Reload DC: 			30%
+Fire modes:			Semi, auto
+Auto fire rate:			8
+Armor pierce level: 		0
+Reflex Modifier: 			+10
+Suppression Level: 		Medium
+
+=====	LEVEL 5 REQUIREMENT WEAPONS	=====
+
+----G&W XMP	$1536
+Damage:	55
+Ranged Miss Chance:
+	4 out to 10m
+	5 out to 20m
+	6 out to 40m
+Burst Miss Chance:		+4 - 1 for every 4 Dex
+Automatic Miss Chance:	+6 - 1 for every 4 Dex
+Mag Size:	30 rounds of 10.0x25mm. $19 per mag
+Reload DC:				30%
+Fire Modes:			Semi, Burst, Auto
+Burst Fire Rate:			3
+Auto fire rate:			6
+Armor Pierce Level:		2
+Reflex Modifier:			+10
+Suppression Level:		Medium
+
+---- Anvil Arms AR-34 PDW	$1356
+Damage:	45
+Ranged Miss Chance:
+	3 out to 10m
+	4 out to 25m
+	5 out to 45m
+Automatic Miss Chance:	+3 - 1 for every 8 Dex
+Mag Size:	15 rounds of 5.00x37mm. $7 per mag
+Reload DC:				30%
+Fire Modes:			Semi, Auto
+Auto fire rate:			6
+Armor Pierce Level:		3
+Reflex Modifier:			10
+Suppression Level:		Medium
+
+----Flexcorp LS55	$2183
+Damage:	35
+Ranged Miss Chance:
+	3 out to 15m
+	4 out to 40m
+	5 out to 50m
+Automatic Miss Chance:	+3 - 1 for every 8 Dex
+Mag Size:	40 rounds of 10kW (7x20mm). $23 per mag

```

Comment:
40 rounds of 10kW. $23 per mag

I forgot to remove the theoretical cartridge size for lasers and plasma (7.00x20mm, etc.), that is a dev note.